### DISCUSSION TITLE
Can we overfit batches if labels are incorrect?

### DISCUSSION THREAD

**Discussion by talhaanwarch** [Lightning-AI/pytorch-lightning#12027] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I want to know can we use `overfit batches` to check the sanity of my labels data? For example if i randomly define the labels or if i shuffle the labels with out shuffling corresponding features, can we still overfit_batches and get 100% results?

**Comment by rohitgr7** [comment]
up to you, but you shouldn't use incorrect labels with overfit_batches, since the idea here is to check whether the model/data pipeline is good enough for further training or not. If you pass in in-correct labels, then the results won't signify what we are trying to infer.
